Sources/MapboxMaps/Foundation/BaseMapView.swift

```diff
@@ -32,6 +32,15 @@ open class BaseMapView: UIView {
 
     private let mapClient = DelegatingMapClient()
 
+    public var options = RenderOptions() {
+        didSet {
+            let defaultPrefetchZoomDelta: UInt8 = 4
```

Comment:
Is this a known constant that needs to be used outside of this scope? If so, we shouldn't make the variable available, only in the didSet local scope. 

Also, what is the context around the value of 4? Can we add some documentation around this please?

Reply:
There's a ticket open for this.